### DISCUSSION TITLE
Badge.Default.SetCount(0)

### DISCUSSION THREAD

**Discussion by douglasc211** [CommunityToolkit/Maui#2406] [Q&A] [answered]
.net 8   Maui toolkit 9.1.1  C#   

After sending a notification to Android phone, the user opens the app and refreshes a list.  refresh makes this call to clear the screen icon badge of counters or notification indicators like a dot.
 
Badge.Default.SetCount(0)

The works for iOS but not for Android.

**Comment by bijington** [comment]
We don't support all options on Android. In fact I think you need to do some work to support it based on our documentation:

https://learn.microsoft.com/dotnet/communitytoolkit/maui/essentials/badge?tabs=android#android-1

**Reply by douglasc211** [reply, marked as answer]
Thank you, I did not read the complete page originally.  Apologies
